contribs/gmf/src/services/permalink.js

```diff
@@ -511,6 +522,35 @@ gmf.Permalink.prototype.getMapCrosshair = function() {
 };
 
 
+/**
+ * Sets the map crosshair to the center (or the map center if nothing provided).
+ * Overwrites an existing map crosshair.
+ * @param {?ol.Coordinate|undefined} center Optional center coordinate.
```

Comment:
Instead of having `{?ol.Coordinate|undefined}` use `{?ol.Coordinate=}` an rename the parameter to `opt_center`.
This is the standard way for optional parameter with the closure compiler.